From 24960464e8f34597ba3ef0cbec10f6ca3d1fa68a Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 6 Mar 2013 14:53:05 +0800
Subject: [PATCH 169/304] ENGR00252404 Make camera selection to be one of many

The selection can be as below setting in init.freescale.rc, which
will make the uvc as first priority:
setprop front_camera_name uvc,ov5642_camera,ov5640_camera

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 sabresd_6dq/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 8b2a1f3..bf91732 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -54,7 +54,7 @@ on boot
     #the android before honycomb are all in portrait mode
     setprop back_camera_name ov5640_mipi
     setprop back_camera_orient 0
-    setprop front_camera_name ov5642
+    setprop front_camera_name uvc,ov5642_camera,ov5640_camera
     setprop front_camera_orient 0
 
     # Set OpenGLES version
-- 
1.8.0

